### PR TITLE
Shrink acceptance test data - PR5

### PR DESCRIPTION
**Pull request by bayliffe**
A component of #1804

To reduce the data volume of the acceptance test data repository, work has been done to coarsen the resolution of test data relating to the following CLIs:

- standardise
- blend-adjacent-points
- lightning-from-cape-and-precip
- hail-fraction
- apply-bias-correction

This PR updates the checksums.
The relevant acceptance test data can be found in branch `shrink_atd_5`
Plotting of the original and modified data [can be seen here](https://github.com/MetOffice/improver_aux/blob/acceptance_test_data_shrinking/notebooks/acceptance_test_data/shrink_atd_5.ipynb).

### CODE REVIEW

**Review by brhooper — APPROVED** [review]
This looks okay.

File: improver_tests/acceptance/SHA256SUMS [hunk 1]
```diff
@@ -577,18 +577,18 @@ eff5c671c33c3520e9e7f0bcab4fe2a4eb5557fd84767da1a8c494f3ab7f3aa6  ./spot-extract
...
 d4404df3a8acdeed27f20ec621a0d3500bf4c849eaf7a7064c4e42f079b37a43  ./standardise/metadata/metadata.json
 326ae7b6d3cff0fd3da840b643816d897e7133a248e49eacad3ec37af669f49e  ./standardise/metadata/radar_metadata.json
 d89a8587bc28b574b8b1e624bb3bf339aaeb9c8c2355db7fe518af8c9bce527c  ./standardise/radarnet/input_coverage.nimrod
 6335cba81be74577fe10d0a4f5cb75724abcef16499d76c263e80c16cf092a05  ./standardise/radarnet/input_preciprate.nimrod
 8a0cc7c53513fe11919da6ae39e431fde7f5b5831e255d8f30feda2a53eb3c71  ./standardise/radarnet/kgo_coverage.nc
 8e4ca14de258f9a511c9ff82ac7df5cc95659bc41c3e745083f5ed449f245d00  ./standardise/radarnet/kgo_preciprate.nc
-4a6d679776a8f7ac35dff33093e1558a2b497527d1537b6bdc5d325bca29097b  ./standardise/stage-v110/input.nc
-575151096829ed458ab13702129efc88f48a8ec459f9b3f39e11e4f5b2e296b2  ./standardise/stage-v110/kgo.nc
+44cf0fd7d6a4d24546da814d08f87f58d2f83ef5ae3a94c4272a4d33659c3f24  ./standardise/stage-v110/input.nc
+add67f9b08aade71cac803044d650a966698820b8e889468b760edaf3bf89dcc  ./standardise/stage-v110/kgo.nc
```
Comment (MoseleyS) [inline improver_tests/acceptance/SHA256SUMS:591]: This test is obsolete. Can you delete it please.

**Review by MoseleyS — CHANGES_REQUESTED** [review]

File: improver_tests/acceptance/SHA256SUMS (hunk 1 above)
Comment (bayliffe) [reply improver_tests/acceptance/SHA256SUMS:591]: If we want to delete this test, we should remove the functionality that it is testing. I've created an issue to handle this separately to this data volume reduction exercise: https://github.com/metoppv/mo-blue-team/issues/457

Comment (MoseleyS) [reply improver_tests/acceptance/SHA256SUMS:591]: That's sensible.

**Review by MoseleyS — APPROVED** [review]

**Pull request merged**